conf/autopilot/rotorcraft_control_loop.xml

```diff
@@ -60,6 +60,7 @@
       <exception cond="RCLost()" deroute="FAILSAFE"/>
     </mode>
 
+<!--
     <mode name="ATTITUDE_Z_HOLD" shortname="A_ZH">
```

Comment:
Not sure how this works and does this affect other drones?

Reply:
The purpose of this autopilot file is the MODULE mode, so their is no problem to remove this, I should have done that in the first place.
If someone think it is still useful, you can do something like this: https://github.com/paparazzi/paparazzi/commit/27860ffa4970d1e3685e78d4dbb87b82221ae6cd#diff-48f82a8991c7345f6c46ad66d35c26c069d40a5a3665f536ddf7e2b652b53057

Reply:
> Not sure how this works and does this affect other drones?

I did a grep and only bebop opticflow airframes use this.

Reply:
> The purpose of this autopilot file is the MODULE mode, so their is no problem to remove this, I should have done that in the first place. If someone think it is still useful, you can do something like this: [27860ff#diff-48f82a8991c7345f6c46ad66d35c26c069d40a5a3665f536ddf7e2b652b53057](https://github.com/paparazzi/paparazzi/commit/27860ffa4970d1e3685e78d4dbb87b82221ae6cd#diff-48f82a8991c7345f6c46ad66d35c26c069d40a5a3665f536ddf7e2b652b53057)

There are now two options that both have the same conditions of mode==1

Reply:
Then instead of commenting just delete it?